clients/cloud/scala/README.md

```diff
@@ -0,0 +1,108 @@
+# Scala Producer and Consumer for Confluent Cloud
+
+This directory includes projects demonstrating how to use the Scala producer and consumer with Confluent Cloud.
+
+For more information, please see the [application development documentation](https://docs.confluent.io/current/api-javadoc.html)
+
+
+# Quickstart
+
+Before running the examples, **you should setup the Confluent Cloud CLI** by running the [quickstart](https://docs.confluent.io/current/quickstart/cloud-quickstart.html#cloud-quickstart).
+
+
+1. Create a topic called `testtopic`:
```

Comment:
Does Scala support the AdminClient?  If yes, I suggest the code use it instead of CCloud CLI.  For an example of the Java version of the AdminClient (if it's analogous, please see my example : https://github.com/confluentinc/examples/blob/DEVX-555/clients/cloud/java/src/main/java/io/confluent/examples/clients/ProducerExample.java#L47-L61 ).  Note that my example is unreviewed so far, but the idea for Scala could be the same (if supported)

Reply:
Added topic creation if does not exist